README.md

```diff
@@ -64,7 +69,24 @@ This example shows the configuration for the [Uploadex.FileStorage](https://hexd
 
 *Note: To avoid too much metaprogramming magic, the `use` in this module is very simple and, in fact, optional. If you wish to do so, you can just define the `@behaviour Uploadex.Uploader` instead of the `use` and then call all lower level modules directly, passing your Uploader module as argument. The `use` makes life much easier, though!*
 
-### 2: Schema
+### 2: Ecto Migration
+
+A string field is required in the database to save the file reference.
+The example below shows what would be needed to have a field to upload.
+
+```elixir
+  defmodule MyRepo.Migrations.AddPhotoToUsers do
```

Comment:
small detail here, but it should be something like `MyApp.Repo.Migrations.AddPhotoToUsers`, right?

Also, the indentation is off